AndroidSDK/src/com/leanplum/activities/LeanplumActivity.java

```diff
@@ -27,6 +27,7 @@
 import com.leanplum.Leanplum;
 import com.leanplum.LeanplumActivityHelper;
 
+@Deprecated
```

Comment:
For all the activities, please add: 
Please add a comment to the javadoc like: "Deprecation: This class will be removed in a future major release. Please use LeanplumLifecyclehelper to track your activities automatically."